source/features/notifications-filter.tsx

```diff
@@ -1,7 +1,17 @@
 import React from 'dom-chef';
 import select from 'select-dom';
 import * as pageDetect from 'github-url-detection';
-import {CheckIcon, CheckCircleIcon, XCircleIcon, IssueOpenedIcon, GitPullRequestIcon, GitPullRequestDraftIcon, GitMergeIcon, DotIcon, DotFillIcon} from '@primer/octicons-react';
+import {
+	CheckIcon,
```

Comment:
Nitpick can  you sort it from a-z. I think the list will be easier to read. @fregante?